esrally/driver/driver.py

```diff
@@ -354,8 +354,7 @@ def _after_track_prepared(self):
         self.send(
             self.start_sender,
             PreparationComplete(
-                # older versions (pre 6.3.0) don't expose build_flavor because the only (implicit) flavor was "oss"
-                cluster_version.get("build_flavor", "oss"),
```

Comment:
This breaks the `ClusterEnvironmentInfo` telemetry device when no build flavor is set, eg. `esrally race --revision=latest --track=geonames --test-mode --car=4gheap,basic-license` breaks.